CONTRIBUTING.md

```diff
@@ -113,6 +113,31 @@ If you'd like to contribute example `.bicep` files that showcase abilities of th
 
 **Note:** If you have never submitted a Pull Request or used git before, reading through the [Git tutorial](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/git-tutorial.md) in the azure-quickstart-template repo is a good place to start.
 
+### Snippets
+
+If you'd like to contribute to the collection of snippets:  
```

Comment:
I was wondering if it's worth laying out some guidelines on what sort of snippets we'd take and around naming. I'm not exactly sure what that would look like, but e.g. I wouldn't want people submitting huge files or highly specific scenarios as snippets.

Reply:
Good call. Added snippet criteria at the top of the guide.